flash

```diff
@@ -664,6 +676,14 @@ mount_boot_disk "${disk}" "${boot}"
   if [ -f "${USER_DATA}" ]; then
     echo "Copying cloud-init ${USER_DATA} to ${boot}/user-data ..."
     cp "${USER_DATA}" "${boot}/user-data"
+    # Hack to set date in a reasonable range to make apt-get update work
+    if ! grep -q "^bootcmd:.*" "${boot}"/user-data; then
+      # USER_DATA doesn't have a bootcmd section, add it
+      echo "" >> "${boot}"/user-data
```

Comment:
Do we really need to always modify he user-data file? 
I prefer to be able to just copy a yaml without magic modifications. 
Maybe activate this feature with a command line option?

Reply:
I think the workaround should be enabled by default and we add an command line option to turn it off. Because without the workaround most install features don't work and the workaround does not break the users config. Only when the user it's self want to set the datetime then the workaround must be disabled.

Reply:
@Legion2

> I think the workaround should be enabled by default...
I think that makes a lot of sense, especially since on RPi 4, the current implementation is dysfunctional (`apt-get update` will fail systematically which in turn makes package installation fail)

Reply:
@StefanScherer 

> I prefer to be able to just copy a yaml without magic modifications.

Is there a mechanism to pick-up a custom yaml created for this purpose?